1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -1,5 +1,6 @@
 export const fizzbuzz = (num: number): string => {
   if (num > 100) throw new Error('Number must be less than 100');
+  if (num < 0) throw new Error('Number must be greater than 0');
```

Comment:
**question**: The check might needs to be `num <= 0` or `num < 1` as the number should be greater then 0.

Reply:
Yes, Thanks!
It accepts numbers between 1 and 100, so it should be < 1